webapp/src/components/modals/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -9,6 +9,7 @@ import FormButton from 'components/form_button';
 import Input from 'components/input';
 
 import GithubIssueSelector from 'components/github_issue_selector';
+import {getErrorMessage} from '../../../utils/user_utils';
```

Comment:
Can we use this import syntax for this import? Same with the import in `create_issue.jsx`
```suggestion
import {getErrorMessage} from 'utils/user_utils';
```